payload.json

```diff
@@ -0,0 +1,5 @@
+{
+  "action": "opened",
+  "number": 2,
+  "pull_request": {}
+}
```

Comment:
what's this file?

Reply:
oops, was using it to test actions locally with act and commited it by mistake